docs/docsite/rst/collections_guide/collections_using_playbooks.rst

```diff
@@ -18,7 +18,7 @@ Once installed, you can reference a collection content by its fully qualified co
 
 This works for roles or any type of plugin distributed within the collection:
 
-.. code-block:: yaml
+.. code-block:: ini
```

Comment:
Shouldn't this be `yaml+jinja`? This is definitely **not** an INI file.

Reply:
Exactly.